subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -31,6 +32,7 @@ enum fp_state {
 	FP_STATE_INITIAL,
 	FP_STATE_USE_TEMP_KEY,
 	FP_STATE_WAIT_FOR_ACCOUNT_KEY,
+	FP_STATE_WAIT_FOR_PERSONALIZED_NAME,
```

Comment:
I think we could consider witching to using bitmask to express the `wait_for_...` states at some point (I think it may be better approach in the long run, since Fast Pair spec informs about marking key as capable for decrypting given type of message)

Reply:
Ok, then let's leave `wait_for_...` states. I would still suggest to use a separate boolean (or bit) to indicate that the used key is an Account Key.